tools/wrangling/xlsx2tsv.xml

```diff
@@ -0,0 +1,50 @@
+<tool id="xlsx2tsv" name="Excel to Tabular" version="0.1.0+galaxy0" license="MIT" profile = "23.0">
+    <description>with pandas</description>
+    <requirements>
+        <requirement type="package" version="2.2.1">pandas</requirement>
+        <requirement type="package" version="3.1.5">openpyxl</requirement>
```

Comment:
Is this an optional requirement of pandas, needed to read xls files?

Reply:
yes, the openpyxl is require :)